app/src/main/java/com/asfoundation/wallet/ui/iab/LocalPaymentFragment.kt

```diff
@@ -292,7 +304,14 @@ class LocalPaymentFragment : DaggerFragment(), LocalPaymentView {
     return inflater.inflate(R.layout.local_payment_layout, container, false)
   }
 
-  override fun getOkErrorClick() = RxView.clicks(error_view.activity_iab_error_ok_button)
+  override fun getOkErrorClick(): Observable<Any> {
+    return RxView.clicks(error_view.error_dismiss)
+  }
```

Comment:
method is called okerror clicks, but it has error_dismiss view. 
why remove the "="?